08-power-quality-monitor/drwattson-build.md

```diff
@@ -0,0 +1,200 @@
+
+We didn't take a photo of this, although you can see the tubing in the next step.
+
+![](images/wiring-05-heat-shrink-tubing.jpg)
+
+### Step 6: Thread the Neutral Wire
+
+Thread the white neutral wire through the current transformer on the Dr. Wattson board.
+
+![](images/wiring-06-neutral-through-current-transformer-above.jpg)
+![](images/wiring-06-neutral-through-current-transformer.JPG)
+
+### Step 7: Join Wires
+
+This step, you will join the wires from the outlet for USB power, wires to power for the Dr. Wattson board and wires from the inlet cable. At this stage, joning can be done by gently twisting the wires together - they will later be soldered in place.
+
+The wires joined all have the same electrical function, neutral is joined to neutral and live to live.
+
+#### Earth
+The green protective earth cable isn't joined yet, and is left as is. We will do this later.
+
+#### Neutral
+
+1. Join the white neutral wire from the inlet cable (the one that passes through the current transformer), with the color-coded neutral 18-gauge power wire (shown as black in the photo below.)
+
+2. Join the white neutral wire from the 18-gauge USB outlet cable to the join made in Step 1.
+
+Once this is done, you will have 3 wires joined:
+
+1. the neutral from the inlet cable, after it passes through the current transformer
+2. the neutral wire used to power the monitoring board
+3. the neutral wire to the USB power outlet
+
+#### Live
+
+1. Join the black live wire from the inlet cable with the color-coded live 18-gauge power wire (shown as red in the photo below.)
+
+2. Join the black live wire from the 18-gauge USB outlet cable to the join made in Step 1.
+
+Once this is done, you will have 3 wires joined:
+
+1. the live from the inlet cable
+2. the live wire used to power the monitoring board
+3. the live wire to the USB power outlet
+
+![](images/wiring-07-usb-receptacle-and-monitor-power.jpg)
+
+### Step 8: Join the primary load outlet cable
+
+In this step, the neutral, live and earth wires from outlet cable are joined to the corresponding wires from the inlet.
+
+Using a soldering iron, solder the green earth wire from the inlet cable to the green earth wire on the outlet cable, as shown in the image below
+
+![](images/wiring-08-outlet-earth.jpg)
+
+Repeat for the neutral wire, soldering the neutral wire of the outlet cable to the neutral join made in step 7.
+
+Repeat once more for for the live wire, soldering the live wire of the outlet cable to the live join made in step 7.
+
+![](images/wiring-08-outlet-neutral-live.jpg)
+
+### Step 9: Apply Heat
+
+With all the electrical joins complete, we can now apply heat to shrink the heat shrink tubing over the joints on the neutral, live and ground joins.
+
+1. gently slip the tubing over the joints
+2. using a heat gun, heat up the heat shrink tubing until it starts to contract
+3. rotate the heat gun around the tubing so that it shrinks on all sides
```

Comment:
There are all full sentences so I’d capitalize the first letter of each and add periods to the ends.